get_notebook.py

```diff
@@ -0,0 +1,8 @@
+import os
+import galaxy
+import shutil
+
+hid = os.environ.get('DATASET_HID', None)
+if hid not in ('None', None):
+    galaxy.get(int(hid))
+    shutil.copy('/import/%s' % hid, '/import/ipython_galaxy_notebook.ipynb')
```

Comment:
There has to be something cleaner we can do. This and the lack of Volumes used in the galaxy fixes for running on test were a bit .. odd/uncomfortable to me. It works, sure, but I think we need to work on cleaning up some of this cruft.